components/Modal/index.d.ts

```diff
@@ -27,14 +24,14 @@ export interface ModalProps {
   };
 }
 
-export default class Modal extends Component<ModalProps> {
-  static Header: ModalHeader;
-
-  static Title: Title;
-
-  static HeaderText: HeaderText;
```

Comment:
Is "ModalText" still used in some scenario? If not, remove the type

Reply:
You are pointing to the version that hasn't changed yet